src/test/java/test_login_use_case/TestLoginUseCase.java

```diff
@@ -0,0 +1,116 @@
+import use_cases.login_usecase.loginResponseModel;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TestLoginUseCase {
+    /**
+     * Test if a user already exists
+     */
+    @Test
+    void existsTest() {
+
+        DatabaseGateway testingDatabase = new Database("Test_Storage");
+
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                assertEquals("demetriusjr", responseModel.getCommonUser().getName());
+                assertTrue(testingDatabase.hasKey("demetriusjr"));
+                testingDatabase.deleteStorageFile();
+                return null;
+            }
+
+            @Override
+            public loginResponseModel prepareFailView(String error) {
+                assertEquals(error, "Account does not exist");
+                return null;
+            }
+        };
+
+        FridgeFactory testingFridgeFactory = new CommonFridgeFactory();
+        ArrayList<CommonIngredient> ingredients = new ArrayList<>();
+        CommonFridge testFridge = (CommonFridge) testingFridgeFactory.create(ingredients);
+        UserFactory testingUserFactory = new CommonUserFactory();
+        CommonUser testUser = (CommonUser) testingUserFactory.create("demetriusjr", testFridge);
+        StorageObjects objects = new StorageObjects(testUser, testFridge);
+        testingDatabase.store("demetriusjr", objects);
+
+        loginInputBoundary testing_login = new LoginInteractor(testingDatabase, testingPresenter);
+        //loginResponseModel inputData = new loginResponseModel(testFridge ,testUser);
+        loginRequestModel inputData = new loginRequestModel("demetriusjr");
+        testing_login.create(inputData);
+    }
+
+    /**
+     * Check if user does not exist program gives expected result
+     */
+    @Test
+    void nonexistentTest(){
+
+        DatabaseGateway testingDatabase = new Database("Test_Storage");
+
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                assertEquals("demetriusjr", responseModel.getCommonUser().getName());
+                assertTrue(testingDatabase.hasKey("demetriusjr"));
+                testingDatabase.deleteStorageFile();
+                return null;
+            }
+
+            @Override
+            public loginResponseModel prepareFailView(String error) {
+                assertEquals(error, "Account does not exist");
+                return null;
+            }
+        };
+        loginInputBoundary testing_login = new LoginInteractor(testingDatabase, testingPresenter);
+        loginRequestModel inputData = new loginRequestModel("demetriusSr");
+        testing_login.create(inputData);
+
```

Comment:
I would suggest deleting the database storage file after every test. This avoids tests possibly interfering with other tests if we create more later.